# EmgArray message for the Thalmic Myo, which has 8 EMG sensors 
# arranged around the arm
# There is a moving field that's unclear what it is looks like a bitmask

int16[8] data
int16 moving
